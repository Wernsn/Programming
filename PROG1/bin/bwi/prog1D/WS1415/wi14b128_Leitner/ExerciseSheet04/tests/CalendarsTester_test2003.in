20
1582 10 16
