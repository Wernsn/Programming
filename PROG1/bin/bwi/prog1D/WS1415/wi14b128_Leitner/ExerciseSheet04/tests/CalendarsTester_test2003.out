statistics for
Saturday, October 16th 1582
1582 is not a leap year
October 1582 has 16 days
it is day 1 of year 1582
it is in week 1 of year 1582
